### DISCUSSION TITLE
SvelteUI website source code for examples

### DISCUSSION THREAD

**Discussion by germgerm** [svelteuidev/svelteui#487] [General]
Is the SvelteUI website source code available ?
Running into situations where copying the displayed example code does not produce the same results when run locally
Would like to see what the actual code behind the examples is
I assume that a missing a prerequisite but but not sure what it is.

For example : Popper

What we see on the SvelteUI site  
![site](https://github.com/svelteuidev/svelteui/assets/1863991/76bb8643-a8ab-4dff-8208-95f644fa995f)

What we get locally
![local](https://github.com/svelteuidev/svelteui/assets/1863991/15467b23-aa99-47d0-a864-3874df2e8786)

Notice that the arrow div is missing in the local version, plus the additional inline styling

**Comment by germgerm** [comment]
I found it - very much missing info from the code displayed on the website

**Comment by BeeMargarida** [comment]
Hi! You can see the code examples inside @svelteuidev/demos package. We have a ticket to include a link to these demos, but it has not been the priority for now.
We do not include full code examples since they can become quite verbose.
